index.d.ts

```diff
@@ -8,17 +8,15 @@
 // Usage:
 // const mockStore = configureStore();
 
-declare module "redux-mock-store" {
-    import {Store} from 'redux';
+import {Store, Middleware, Unsubscribe} from 'redux';
 
-    interface MockStore extends Store {
-        getState(): any;
-        getActions(): Array<any>;
-        dispatch(action: any): any;
-        clearActions(): void;
-        subscribe(): any;
-    }
-
-    function configureStore(...args: any[]) : (...args: any[]) => MockStore;
-    export = configureStore;
+interface MockStore<S> extends Store<S> {
+    getState():S;
+    getActions():Array<any>;
+    dispatch(action:any):any;
```

Comment:
Yeah sorry, missed those comments. I tend to agree, just use this for now, because it works, we can look at modifying later. Also tests would be nice to prevent breakages in the future!